### ISSUE TITLE
ft2nfdump - Not found

### ISSUE THREAD

**Issue by BaptisteBemel** [phaag/nfdump#453]
It seems to me that ft2nfdump is no longer installed correctly when the nfdump package is installed. Whether it's on an old debian 10 which still allows flow-tools or a more recent version, ft2nfdump is not installed. However, it used to install properly.

**Comment by phaag** [comment]
Please make sure you run configure with the appropriate option:

`./configure .. --enable-ftconv`

and optionally with `--with-ftpath=/path/to/flow-tools/installed` if configure does not find the library or header files.

Then the installation should work as expected. There was no change for this option and it still works.

Hope, this helps

**Comment by BaptisteBemel** [comment]
For some reason, the installation went right when done a first time and the second time I did it, ft2nfdump was not installed with the rest of the files. It seems that, nfdump does not find the headers of flow-tools. However, flow-tools does not have a libft.so but only a libft.a . It can be reinstalled by extracting flow-tools, etc etc. I found another solution that does not need ft2nfdump and did not continue on this path. Thank you for the fast reply to the issue.
